hosts

```diff
@@ -18,3 +18,6 @@ localhost
 
 [buildserver]
 54.89.31.80
+
+[launched]
+34.236.154.139
```

Comment:
Umm.. do you want to commit [launched] that is a dynamically generated list.

Reply:
Whoops, fixed.